main/design_documents/__ARM_FEATURE_MOPS.rst

```diff
@@ -0,0 +1,75 @@
+=============================================================
+Introducing extensions for memcpy family of memory operations
+=============================================================
+
+Following the announcement of the `2021 Architecture Extensions
+<https://community.arm.com/developer/ip-products/processors/b/processors-ip-blog/posts/arm-a-profile-architecture-developments-2021>`_,
+I'd like to propose these aditions to the ACLE to support the new instructions
```

Comment:
Maybe avoid addressing this in first person, but use a generic "we"?

`We are proposing these additions to the ...`